recipes/en_core_web_sm/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set prefix = "spacy-model-" %}
+{% set name = "en_core_web" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ prefix }}{{ name|lower }}
+  version: {{ version }}
```

Comment:
Do you want all outputs to have the same version?

Reply:
Yes, in this case all the english models have the same version. I will issue a separate PR for the other languages.

This should be ready to merge if you agree with it.